src/state/user/user-action-creators.js

```diff
@@ -0,0 +1,61 @@
+import * as userApi from '~/src/api/user-api';
+import userActionTypes from './user-action-types';
+import { browserHistory } from 'react-router';
+
+export function usernameChanged(username) {
+    return {
+        type: userActionTypes.USERNAME_CHANGED,
+        data: { username },
+    };
+}
+
+export function passwordChanged(password) {
+    return {
+        type: userActionTypes.PASSWORD_CHANGED,
+        data: { password },
+    };
+}
+
+export function login(username, password, history) {
+    return (dispatch) => {
+        userApi.login(username, password, history)
+            .then((response) => {
+                const { auth_token } = response;
+                console.log(auth_token);
+                localStorage.setItem('auth_token', auth_token);
```

Comment:
Since we are accessing this same local storage in different spots, can we make this a constant?